### PR TITLE
Change old syntax to new one in the illustro skin

### PR DESCRIPTION
**Pull request by nek7u**
* From `Measure=Plugin` to `Measure=RecycleManager`, `Measure=WebParser` (previously a plugin measure)  
* Removed file extension. From `Plugin=InputText.dll` to `Plugin=InputText`

### CODE REVIEW

File: Build/Skins/illustro/Google/Google.ini [hunk 1]
```diff
@@ -34,7 +34,7 @@ colorText=255,255,255,205
 ; However, it's not the easiest way to start learning Rainmeter. Get a
 ; hold of the basics, and then check out the manual entry for InputText.
 Measure=Plugin
-Plugin=InputText.dll
+Plugin=InputText
```
Comment (Joehuu) [inline Build/Skins/illustro/Google/Google.ini:37]: The comment above should also have `.dll` removed.

Comment (nek7u) [reply Build/Skins/illustro/Google/Google.ini:37]: Thank you for the review, as always.  
I have fixed that.

Comment (Joehuu) [reply Build/Skins/illustro/Google/Google.ini:37]: Doesn't seem to be applied. I was talking about this:
https://github.com/rainmeter/rainmeter/blob/55b75a812f531aaf7510c66aef732a785a3e1302/Build/Skins/illustro/Google/Google.ini#L33

Comment (nek7u) [reply Build/Skins/illustro/Google/Google.ini:37]: Done. Thanks!

**Review by Joehuu — APPROVED** [review]

**Comment by poiru** [comment]
Thanks!

**Pull request merged**